src/test/java/com/kenticocloud/delivery/DeliveryParameterBuilderTest.java

```diff
@@ -272,8 +272,17 @@ public void testLanguage() {
     }
 
     @Test
+    public void testLocaleLanguage() {
+        List<NameValuePair> params = DeliveryParameterBuilder.params().language(new Locale("pt-br")).build();
```

Comment:
This test is only setting the language "pt-br" instead of the language "pt" with variant "br".